script/Common/00_02.txt

```diff
@@ -356,17 +356,17 @@ I feel like I'm about to cry.
 [sha:6dabcc2c83d600812aaade42f5d0893f19d198dc]{
 -- Page 16, Offset 79.
 -- 森はくろくて見たくない。
-I didn't want to look at the forest because it was black.
+I don't want to look at the forest because it's black.
 }
 [sha:a6f4180af63ab54d45920f6801aa581f3e3ac4fc]{
 -- Page 16, Offset 80.
 -- 地面はあかくて見たくない。
-And I didn't want to look at the ground because it was red.
+I don't want to look at the ground because it's red.
 }
 [sha:011cc0beec7d1f0946880df68280e4d9856eb83c]{
 -- Page 16, Offset 81.
 -- だから　空を見あげることにした。
-So I raised my gaze to the heavens.
+So I raise my gaze to the heavens.
```

Comment:
A little bit too advanced for lil Shiki.

> So I look up at the sky.

Reply:
this line is kinda tight and is reflected again later, so even if it's a little too advanced I'd rather it stay.

Reply:
Just curious, what do you mean by tight?

Reply:
as in "cool", not as in "compact"

Reply:
You have no idea how hard I felt like a boomer just now

Reply:
I also think this line goes hard